src/components/views/settings/SetIntegrationManager.tsx

```diff
@@ -57,7 +57,7 @@ export default class SetIntegrationManager extends React.Component<IProps, IStat
     };
 
     public render(): React.ReactNode {
-        const ToggleSwitch = sdk.getComponent("views.elements.ToggleSwitch");
+        const LabelledToggleSwitch = sdk.getComponent("views.elements.LabelledToggleSwitch");
```

Comment:
Could we please get rid of this `sdk.getComponent()` while we're here and just use a regular import